js_modules/dagit/packages/core/src/metadata/MetadataEntry.tsx

```diff
@@ -158,7 +158,9 @@ export const MetadataEntry: React.FC<{
     case 'IntMetadataEntry':
       return <>{entry.intValue !== null ? entry.intValue : entry.intRepr}</>;
     case 'BoolMetadataEntry':
-      return entry.boolValue !== null ? <>{entry.boolValue.toString()}</> : null;
+      return <>{entry.boolValue !== null ? entry.boolValue.toString() : 'null'}</>;
```

Comment:
Just want to call out this line in case it was an earlier change / idea - it looks like the BoolMetadataEntry can also now be null? (looks like the previous version actually rendered nothing and not the string null)

Reply:
Pretty sure this should never actually become null, a null for bool metadata value will get caught on the backend. But leaving this in place just in case-- and IMO we should render null the same everywhere.